_pages/security.md

```diff
@@ -3,64 +3,6 @@ title: General Security Standards
-* all top-level URLS (ex: staging.18f.us _and_ 18f.gsa.gov)
-
-If during testing the system performs:
-
-* user authentication or authorization
-* back-end administrative functions
-* encryption
-
-...then those features cannot be "relatively simple" but must in fact be *complete* before an authorization will be given. Note that the use of common web frameworks and [18F HTTPS standards](https://github.com/18F/https) resolves these issues in almost every case.
-
-Lastly, make sure the `README` file in your repo is fully up to date and clearly explains what the system does and why at a high level. You should also include the above information in your `README`.
-
-### Greybox testing
-
-Once you are ready, 18F DevOps and GSA InfoSec will start both automated and manual scanning and testing. This includes:
-
-* using SSH to go into your EC2 servers to run tests on the operating system
-* [web vulnerability scanning](../dynamic-scanning/) on the front-end
-* [static code analysis](static-analysis/) on the `master` branch of your repo
-* white hat hackers attempting to penetrate the system
-
-For greybox testing, the independent testing team has significant (but not necessarily complete) knowledge of how the system works, as opposed to black box testing, where they have zero knowledge, or white box testing, where they have complete knowledge.
-
-All of these tests must be conducted on all environments and stacks, including one _identical to production_. We call this environment and stack "pre-production" and will also affix the designation "scanee" so there is no ambiguity as to what is being scanned.
-
-This also requires a stable `master` branch. You can continue working on `feature` branches and deploy those to a development environment.
-
-### Resolving vulnerabilities
-
-See [Alerts](../security/scanning/#alerts).
-
-### Expectation management
-
-Overall, if *no* vulnerabilities are found, this process has been taking approximately 2 weeks for test preparation and [System Security Plan](../ato/ssp/) writing and 2 weeks for [greybox testing](#greybox-testing) and [signature](../ato/#signing-an-authorization).
-
-Since the time it would take to resolve vulnerabilities is not known until a vulnerability is identified, it is **strongly recommended** that no expectations are set with Federal partners or GSA stakeholders when public testing will begin. Instead, we recommend that the authorization process is seen as part of the delivery process and your definition of "done".
-
-After the system has been authorized, you can then begin planning a public roll out of your test system.
+_**Before reading the rest of this policy**_, you should read the [readme file at FISMA Ready](https://github.com/fisma-ready/fisma-ready.github.io/blob/master/README.md), a community project co-managed by 18F. You should also take a quick glance at the [components that are already FISMA Ready](https://github.com/fisma-ready).
```

Comment:
Also, I don't think project teams need to know about FISMA Ready at this point, since they won't be using any of those components directly.